Source/AssetRipper.GUI.Web/Pages/Commands.cs

```diff
@@ -20,6 +20,15 @@ internal static RouteHandlerBuilder AcceptsFormDataContainingPath(this RouteHand
 		return builder.Accepts<PathFormData>("application/x-www-form-urlencoded");
 	}
 
+	private static bool TryGetCreateSubfolder(IFormCollection form)
+	{
+	    if (form.TryGetValue("CreateSubfolder", out StringValues values))
+	    {
+		    return values == "true";
+	    }
+	    return false;
+	}
+
```

Comment:
You should make sure that all your changes use tabs and lf line endings. I discovered it here, but there are other places too.